### PR TITLE
[goldpinger] Update image tags for version v3.3.0 

### PR DESCRIPTION
**Pull request by ymrsmns**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

Bump up version image
latest version v3.3.0

### CODE REVIEW

File: charts/goldpinger/Chart.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ name: goldpinger
 sources:
   - https://github.com/bloomberg/goldpinger
   - https://github.com/okgolove/helm-charts
-version: 5.0.1
+version: 5.0.2
```
Comment (okgolove) [inline charts/goldpinger/Chart.yaml:14]: Bump minor version please

File: charts/goldpinger/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ This chart bootstraps a [Goldpinger](https://github.com/bloomberg/goldpinger) de
 To install the chart with the release name `my-release`:
 
 ```console
-$ helm install --name my-release stable/goldpinger
+$ helm install --name my-release okgolove/goldpinger
```
Comment (okgolove) [inline charts/goldpinger/README.md:26]: Good catch!

Comment (ymrsmns) [reply charts/goldpinger/README.md:26]: Done

File: charts/goldpinger/Chart.yaml (hunk 1 above)
Comment (ymrsmns) [reply charts/goldpinger/Chart.yaml]: Please review

**Review by okgolove — APPROVED** [review]

**Pull request merged**